kymatio/scattering1d/scattering1d.py

```diff
@@ -692,7 +701,7 @@ def scattering(x, psi1, psi2, phi, J, pad_left=0, pad_right=0,
             S[:, cc, :] = S1_J.squeeze(dim=1)
             cc += 1
         else:
-            S[n1,] = S1_J
+            S[n1, ] = S1_J
```

Comment:
uuuuh ... ? That looks like R code

`S[n1]`

Reply:
oh, ok. i was confused seeing this

Reply:
It's because when `vectorize=True`, the output is a dictionary indexed by tuples of filter indices. Consequently, the zeroth order is in `S{()]`, the first order with filter `n1` is in `S[(n1,)]`, and the second order with filters `n1` and `n2` is in `S[(n1,n2)]`. Not sure if we're going to keep this capability, but that's what's going on.

I'm assuming correct style would call for `S[(n1,)]`?

Reply:
OOOH, my bad! Sorry about  that.

Yeah, so as much as I like to point out that tuples are defined because of commas, not parentheses (i.e. `n1,` is a tuple just fine), I think it's better to have the parentheses here for readability. So yeah `S[(n1,)]` as you say

Reply:
addressed in 89e81fe